src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * 
+ * @author Naren
+ * Association class for keeping track of changes to tags
+ */
+public class Tagging {
+	public static enum function {ADD, REMOVE};
+	
+	public static List<String> ALL_TAGGINGS = new ArrayList<>();
```

Comment:
Based on the class diagram given in the exercise, the list of `Tagging` objects should not be in this class. If it was in this class, the diagram would be like this:
![image](https://cloud.githubusercontent.com/assets/1673303/18409597/69111e4e-777d-11e6-92bf-9914c825bd13.png)